### DISCUSSION TITLE
Difference between ArchWSL and WSLdl?

### DISCUSSION THREAD

**Discussion by i2** [yuk7/wsldl#134] [General]
Hi, Other than the obvious thing that ArchWSL is derived from WSLdl and it's Arch only, what other real differences? Looking at code updates, would there be any chance ArchWSL gets an updates regarding its distro-launcher that it's not ported back to WSLdl?

[I essentially would like to be on the most recent version of WSL distro launcher. My distro would be non-Arch.]

**Comment by yuk7** [comment]
ArchWSL's launcher uses the wsldl binary as is.
There is no difference in the code and no chance that changes made to ArchWSL's launcher will not be incorporated into the wsldl mainstream.
However, ArchWSL includes custom configuration files and images in its packages.
This may alter the default behavior of wsldl.
